workspaces/spectacle/src/spectacle.ts

```diff
@@ -673,10 +723,17 @@ export async function makeSpectacle(opticContext: IOpticContext) {
         // TODO FLEB - connect up to optic engine
         return false;
       },
-      commands: (parent: FieldShape) => {
-        return {
-          remove: commandGenerator.field.remove(parent.fieldId),
-        };
+    },
+    FieldCommands: {
+      remove: async (parent: FieldShape) => {
```

Comment:
This type is actually just `{fieldId: string}` - since only `commands` gets resolved as the parent